src/index.js

```diff
@@ -25,15 +26,25 @@ class DelegatedPeerRouting {
    * Attempts to find the given peer
    *
    * @param {PeerID} id
+   * @param {number} timeout How long the query can take. Defaults to 30 seconds
```

Comment:
Since we are changing this, maybe we should change this param to an `options` object with the `timeout` property, and for now, keep accepting it as a number, but deprecate in the future, in favor of options.

Reply:
Yeah, that sounds good, let's change those up!